src/test/dynamicResources/awsResourceManager.test.ts

```diff
@@ -185,7 +185,7 @@ describe('ResourceManager', function () {
 
     it('registers schema mappings when opening in edit', async function () {
         const editor = await resourceManager.open(resourceNode, false)
-        verify(schemaService.registerMapping(anything())).once()
+        verify(schemaService.registerMapping(anything(), anything())).once()
```

Comment:
This is an example of why mocks are burdensome. These tests failed because of a parameter update.